frontend/src/components/applyForm/ApplyFormSuccessMessage.tsx

```diff
@@ -0,0 +1,11 @@
+import { Alert } from "@trussworks/react-uswds";
+
+export const ApplyFormSuccessMessage = ({ message }: { message: string }) => {
+  return (
```

Comment:
we probably want to clear this message at some point - maybe when the user touches the form after save?

Reply:
I'm not sure I understand your comment. Currently this is shown when a user clicks "save".

Reply:
The main scenario I'm thinking of where confusion could happen if the message isn't cleared is this one

1. user saves the application
2. user sees the save notification
3. user edits the application while the save notification is still present
4. user saves again
5. user sees the save notification
6. has the second save actually occurred or is this still the same message from the first save?

Reply:
The message gets cleared upon save / submit.